packages/aragon-cli/src/commands/run.js

```diff
@@ -6,7 +6,7 @@ const publish = require('./apm_cmds/publish')
 const devchain = require('./devchain')
 const deploy = require('./deploy')
 const newDAO = require('./dao_cmds/new')
-const startIPFS = require('./ipfs')
+const startIPFS = require('./ipfs_cmds/start')
```

Comment:
😅 This one almost slipped... can't wait to have typescript or tests to catch these things.